package.json

```diff
@@ -58,6 +58,23 @@
     "test": "node ./node_modules/vscode/bin/test"
   },
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "command-explorer",
+          "title": "Command Explorer",
+          "icon": "media/dep.svg"
+        }
+      ]
+    },
+    "views": {
+      "command-explorer": [
+        {
+          "id": "commands",
```

Comment:
Could we prefix this with  `PowerShell.` so that we have a common prefix and a better chance of it being unique.

Reply:
Done!